src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -283,6 +262,21 @@ public void writeAssignmentExpr_nullObjectValueReferenceType() {
     assertEquals(writerVisitor.write(), "String x = null");
   }
 
+  @Test
+  public void writeAssignmentExpr_stringObjectValue() {
```

Comment:
Could we please add a basic test case for writing StringObjectValue?

Reply:
cool, added a basic valueExpr with string object value.